services-directions/src/test/java/com/mapbox/api/directions/v5/MapboxDirectionsTest.java

```diff
@@ -823,4 +823,27 @@ public void callStart(Call call) {
 
     assertEquals(eventListener, mapboxDirections.eventListener());
   }
+
+  @Test
+  public void testPost() throws IOException {
+    MapboxDirections mapboxDirections = MapboxDirections.builder()
```

Comment:
Should you add a post fixture to avoid the bug that you found in MapboxMapMatching?